docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -153,6 +153,22 @@ values. Typical usage does not require you to configure them.
     * - ``delta.target-max-file-size``
       - Target maximum size of written files; the actual size may be larger.
       - ``1GB``
+    * - ``delta.immutable-partitions``
+      - Can new data be inserted into existing partitions?
+        If ``true`` then setting
+        ``delta.insert-existing-partitions-behavior`` to ``APPEND``
```

Comment:
Sounds like we're inheriting configuration complexity from Hive, where it seems to be result of a backward compatibility (legacy).

Is this exactly what we want to do here?